packages/Dockerfile

```diff
@@ -306,13 +306,7 @@ COPY --from=ci-qt-builder /package/. /
 FROM ci-python-builder as ci-pyside-builder
 
 # Workaround pyside issue with Qt headers within system location https://bugreports.qt.io/browse/PYSIDE-787
-COPY ../scripts/base/build_qt.sh /tmp/
-RUN --mount=type=cache,target=/tmp/ccache \
-    --mount=type=cache,sharing=private,target=/tmp/downloads \
-    source /tmp/versions_base.sh && \
-    export ASWF_INSTALL_PREFIX=/tmp/qt5temp && \
-    /tmp/build_qt.sh && \
-    ccache --show-stats
+COPY --from=ci-package-qt /. /tmp/qt5temp
```

Comment:
Assuming this is a multi-file copy and that /tmp/qt5temp is a destination directory, shouldn't there be a trailing slash after /tmp/qt5temp/

Reply:
good point, but it seems that the docker `COPY` operation doesn't require that...